relay-event-normalization/src/transactions/processor.rs

```diff
@@ -382,18 +376,17 @@ fn is_high_cardinality_transaction(event: &Event) -> bool {
 ///
 /// Replaces UUIDs, SHAs and numerical IDs in transaction names by placeholders.
 /// Returns `Ok(true)` if the name was changed.
-pub(crate) fn scrub_identifiers(string: &mut Annotated<String>) -> Result<bool, ProcessingAction> {
-    scrub_identifiers_with_regex(string, &TRANSACTION_NAME_NORMALIZER_REGEX, "*")
+pub(crate) fn scrub_identifiers(string: &mut Annotated<String>) {
+    scrub_identifiers_with_regex(string, &TRANSACTION_NAME_NORMALIZER_REGEX, "*");
 }
 
 fn scrub_identifiers_with_regex(
     string: &mut Annotated<String>,
     pattern: &Lazy<Regex>,
     replacer: &str,
-) -> Result<bool, ProcessingAction> {
```

Comment:
This function is only called from the `scrub_identifiers` above and the `Result` is never used, so I removed it instead of adding a `.ok()` to keep an unused return.